tests/unit/amo/pages/TestUserProfileEdit.js

```diff
@@ -152,7 +155,11 @@ describe(__filename, () => {
       screen.getByText('Introduce yourself to the community if you like'),
     ).toBeInTheDocument();
     expect(
-      screen.getByRole('link', { name: 'Manage Firefox Accounts…' }),
+      screen.getByRole('link', {
+        name: isMzaBranding()
+          ? 'Manage Mozilla Accounts…'
+          : 'Manage Firefox Accounts…',
+      }),
```

Comment:
Rather than adding conditionals to each test, I think it's clearer to just mock `isMzaBranding` in the test setup (to either true or false).